addons/languages/italian/pack/src/main/res/xml/italian_qwerty_with_symbols.xml

```diff
@@ -7,40 +7,40 @@
 
     <Row>
         <Key android:codes="q" android:popupCharacters="1¹₁" ask:hintLabel="1" android:keyEdgeFlags="left"/>
-        <Key android:codes="w" android:popupCharacters="2²₂ŵ" ask:hintLabel="2"/>
-        <Key android:codes="e" android:popupCharacters="3€³₃êèéëęē" ask:hintLabel="3"/>
```

Comment:
why did you remove so many popup characters? Usually, the `XXX_with_symbols` layouts have a lot of symbols all around.

Reply:
I thought that I removed that commit but apparently I squashed it. I was supposed to push it after arena commit about reordering popup characters. Those diacritcs don't belong to Italian language, so they need to be removed if arena's commit gets merged.

Reply:
okay. for now, can you get them back?